src/pke/examples/iterative-ckks-bootstrapping.py

```diff
@@ -0,0 +1,119 @@
+from openfhe import *
+import math
+import random
+
+def main():
+    IterativeBootstrapExample()
+
+def CalculateApproximationError(result,expectedResult):
+    if len(result) != len(expectedResult):
+        raise Exception("Cannot compare vectors with different numbers of elements")
+    # using the infinity norm
+    maxError = 0
+    for i in range(len(result)):
+        # error is abs of the difference of real parts
+        error = abs(result[i].real - expectedResult[i].real)
+        if error > maxError:
+            maxError = error
+    # resturn absolute value of log base2 of the error
+    return abs(math.log(maxError,2))
```

Comment:
Shorten and make it more pythonic

max_error = max([abs(el1.real - el2.real) for (el1, el2) in zip(result, expectedResult)])